react-cardknox-ifields/package.json

```diff
@@ -1,13 +1,9 @@
 {
   "name": "@cardknox/react-ifields",
-  "version": "1.0.2",
+  "version": "1.2.60",
   "description": "A React component for Cardknox iFields",
   "license": "MIT",
   "main": "build/index.js",
-  "dependencies": {
-    "prop-types": "^15.8.1",
-    "react": "^18.2.0"
```

Comment:
I believe 'react' is needed (if you don't have react installed globally)

Reply:
Put it back (you told me to revert it...)